server/server.js

```diff
@@ -1,42 +1,40 @@
 require('newrelic');
+const Product = require('../db/index.js');
 const express = require('express');
 const path = require('path');
 const mongoose = require('mongoose');
 const redis = require('redis');
-  redisClient = redis.createClient('redis://127.0.0.1:6379');
 const bodyParser = require('body-parser');
-const Product = require('../db/index.js');
 const cors = require('cors');
-const app = express();
 const responseTime = require('response-time');
 const port = 3001;
 
-app.use(cors());
-app.use(bodyParser.json());
-app.use(express.static(path.join(__dirname, '/../client/public')));
-app.use(responseTime());
-
+const app = express();
+const redisClient = redis.createClient('redis://127.0.0.1:6379');
 
 redisClient.on("error", function (err) {
   console.log(err);
 });
 
+app.use(cors());
+app.use(bodyParser.json());
+app.use(express.static(path.join(__dirname, '/../client/public')));
+app.use(responseTime());
 
-app.get('/api/:id', (req, res) => {
 
-  // console.log(req, 'GET :: server');
 
-  // let db = mongoose.connect('localhost:27017/main');
+app.get('/api/:id', (req, res) => {
   redisClient.get(`info-${req.params.id}`, (error, result) => {
     if (result) {
-      res.status(200).json(data);
+      res.status(200).json(JSON.parse(result));
```

Comment:
Why are you parsing and then stringifying again?

Reply:
It doesn't store the data in Redis if I don't stringify it and I need to parse it so I can use the data.